styles/variables.scss

```diff
@@ -0,0 +1,9 @@
+:root {
+	--color-white: #ffffff;
+	--color-carnation: #f65261;
+	--color-emperor: #555555;
+	--color-black: #000000;
+	--color-mine-shaft: #232323;
+	--color-tundora: #424242;
+	--font-primary: "Montserrat", sans-serif;
```

Comment:
please just add small comments in order to separate types of variables, for example:
```
// color variables
--color-white: #ffffff;
--color-carnation: #f65261;
...
// font variables
--font-primary: "Montserrat", sans-serif;
```